components/common/circular_chart/circular_chart.tsx

```diff
@@ -0,0 +1,51 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+import React from 'react';
+
+import './circular_chart.scss';
+
+type CircularChartProps = {
+    value: number;
+    isPercentage: boolean;
+    width: number;
+    height: number;
+    type: 'info' | 'warning' | 'error' | 'success';
+};
+
+const CircularChart = ({
+    value,
+    isPercentage,
+    width,
+    height,
+    type,
+}: CircularChartProps): JSX.Element | null => {
```

Comment:
we will not return `null` here so it can be removed from the return type

Reply:
ok